server/src/modules/users/application/use-cases/create-user/create-user-use-case.ts

```diff
@@ -60,47 +54,41 @@ export class CreateUserUseCase implements UseCaseWithDTO<CreateUserDTO, CreateUs
     try {
       const userAlreadyExists = await this.userRepo.exists(email)
 
-      if (userAlreadyExists && userAlreadyExists.isOk()){
+      if (userAlreadyExists && userAlreadyExists.isOk()) {
```

Comment:
I'm pretty sure `userAlreadyExists` is always truthy. Did you mean
```suggestion
      if (userAlreadyExists.isOk() && userAlreadyExists.value) {
```